### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by niden**
Code for 1.0.0 release.

- Changed all references from `phalcon4` to `phalcon5`
- Made necessary changes to namespaces for `Di` and `Url` since those classes were moved to new locations

Tests are failing because the CI expects phalcon-5.0.0beta1. Once this is released, we can release beta1 and then the tests will pass.

### CODE REVIEW

**Comment by niden** [comment]
@DavertMik When you get a chance please have a look at this.

File: tests/unit/Phalcon4ModuleTest.php [hunk 1]
```diff
@@ -0,0 +1,192 @@
+<?php
```
Comment (Jeckerson) [inline tests/unit/Phalcon4ModuleTest.php]: Name of the file is incorrect

File: tests/unit/Phalcon4ConnectorTest.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
+<?php
```
Comment (Jeckerson) [inline tests/unit/Phalcon4ConnectorTest.php]: Name of the file is incorrect

**Review by Jeckerson — CHANGES_REQUESTED** [review]

File: tests/unit/Phalcon4ConnectorTest.php (hunk 2 above)
Comment (niden) [reply tests/unit/Phalcon4ConnectorTest.php]: Corrected

File: tests/unit/Phalcon4ModuleTest.php (hunk 1 above)
Comment (niden) [reply tests/unit/Phalcon4ModuleTest.php]: Corrected

**Review by Jeckerson — APPROVED** [review]

**Pull request merged**